mappings/net/minecraft/block/BlockDoor.mapping

```diff
@@ -40,6 +40,7 @@ CLASS amb net/minecraft/block/BlockDoor
 		ARG 6 hitX
 		ARG 7 hitY
 		ARG 8 hitZ
+	METHOD a setOpened (Laiw;Lcn;Z)V
```

Comment:
This method sets the open state of the door. The params are world, position, and whether or not you want the door to be open. `setOpened` sounds right to me, but `setOpen` could also work.